src/components/Panel/index.stories.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { H4 } from '../Heading'
+import { BaseButton } from '../BaseButton'
+import { Toggle } from '../Toggle'
+import { Panel, PanelContent, PanelText, PanelBody, PanelFooter } from './'
+
+const Wrapper = story => <div style={{ padding: 24 }}>{story()}</div>
+
+export default {
+  title: 'Panel',
+  decorators: [Wrapper],
+}
+
+export const Single = () => (
+  <Panel>
+    <PanelContent>
+      <PanelBody>
+        <H4>Title</H4>
+        <PanelText>Lorem ipsum dolor sit amet</PanelText>
+        <BaseButton>Edit</BaseButton>
+      </PanelBody>
+    </PanelContent>
+  </Panel>
+)
+
+export const Group = () => (
+  <Panel>
+    <PanelContent>
+      <PanelBody>
+        <H4>Title</H4>
+        <PanelText>Lorem ipsum dolor sit amet</PanelText>
+        <BaseButton>Edit</BaseButton>
+      </PanelBody>
+      <PanelFooter>
```

Comment:
Footer? 😛

Reply:
Haha yeahhh, I know. Been thinking about that one for quite a while. Had `PanelAdornment` at first, but it’s not really and "adornment". I think it makes more sense if you look at it this way:

<img width="540" alt="Screenshot 2019-09-26 at 20 44 53" src="https://user-images.githubusercontent.com/491962/65716034-a3425a00-e09e-11e9-94dc-485b880fd2fe.png">